<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mySnifferGUIClass</class>
 <widget class="QMainWindow" name="mySnifferGUIClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>945</width>
    <height>746</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mySnifferGUI</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,5">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="groupBoxLayout">
       <item>
        <widget class="QGroupBox" name="adapterGroup">
         <property name="title">
          <string>网卡</string>
         </property>
         <layout class="QVBoxLayout" name="adapterGroupLayout">
          <item>
           <widget class="QComboBox" name="adapterBox"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="startAndStopLayout">
            <item>
             <widget class="QPushButton" name="startBtn">
              <property name="text">
               <string>开始</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stopBtn">
              <property name="text">
               <string>暂停</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="IPGroup">
         <property name="title">
          <string>IP</string>
         </property>
         <layout class="QVBoxLayout" name="IPGroupLayout">
          <item>
           <widget class="QLineEdit" name="IPInput"/>
          </item>
          <item>
           <widget class="QPushButton" name="actIPBtn">
            <property name="text">
             <string>筛选</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget"/>
      <widget class="QTreeWidget" name="treeWidget">
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>945</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuwenjian">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action1"/>
    <addaction name="separator"/>
    <addaction name="action2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>设置</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>关于</string>
    </property>
   </widget>
   <addaction name="menuwenjian"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action1">
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action2">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mySnifferGUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>adapterBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>mySnifferGUIClass</receiver>
   <slot>on_adapterBox_currentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tableWidget</sender>
   <signal>cellClicked(int,int)</signal>
   <receiver>mySnifferGUIClass</receiver>
   <slot>on_tableWidget_cellClicked(int,int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>536</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>769</x>
     <y>-2</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_adapterBox_currentIndexChanged(int)</slot>
  <slot>on_tableWidget_cellClicked(int,int)</slot>
 </slots>
</ui>
